package.json

```diff
@@ -13,9 +13,11 @@
     "@credify/nodejs": "^1.21.0",
     "body-parser": "^1.20.0",
     "express": "^4.18.1",
-    "morgan": "^1.10.0"
+    "morgan": "^1.10.0",
+    "axios": "^1.0.0"
   },
   "devDependencies": {
+    "dotenv": "^16.0.3",
```

Comment:
dotenv shoudn't be in `dependencies`?

Reply:
I fixed it  :D.

Reply:
Did you update yarn.lock? I mean just updating package.json is not enough. Remove node_modules and yarn.lock, and run `yarn` once again.

Reply:
It fixed it